codebuild/release/publish.yml

```diff
@@ -4,6 +4,8 @@ env:
   variables:
     NODE_OPTIONS: "--max-old-space-size=4096"
     BRANCH: "master"
+    # An explicit distribution tag
+    DIST_TAG: "latest"
```

Comment:
Question: Can this be overridden at some point before the release?
I think that is how CodeBuild works... but I just want to verify.